satpy/tests/reader_tests/modis_tests/test_modis_l1b.py

```diff
@@ -77,6 +77,10 @@ def _load_and_check_geolocation(scene, resolution, exp_res, exp_shape, has_res,
         assert lat_arr.shape == exp_shape
         # compute lon/lat at the same time to avoid wasted computation
         lon_vals, lat_vals = dask.compute(lon_arr, lat_arr)
+        assert lon_arr.dtype == lat_arr.dtype
+        assert lon_arr.dtype == np.float32
+        assert lon_vals.dtype == lon_arr.dtype
+        assert lat_vals.dtype == lat_arr.dtype
```

Comment:
I think it would be clearer (not by much though) with:
```suggestion
        assert lon_arr.dtype == np.float32
        assert lon_arr.dtype == np.float32
        assert lon_vals.dtype == np.float32
        assert lat_vals.dtype == np.float32
```

Reply:
I'm OK changing this, but I do feel like there is a difference in intention here. In my original code I was trying to explain the individual cases:

1. Do two DataArrays have the same dtype
2. Is a single DataArray 32-bit float
3. Is the numpy array the same as the DataArray
4. Is the other numpy array the same as the other DataArray

Reply:
No problem, you can leave it as it is.